reporter/src/funTest/assets/evaluated-model-reporter-test-expected-output.json

```diff
@@ -22,10 +22,10 @@
   } ],
   "licenses" : [ {
     "_id" : 0,
-    "id" : "Apache-2.0 (multi-line)"
+    "id" : "LicenseRef-test-Apache-2.0-multi-line"
```

Comment:
Commit message nit: These are not SPDX IDs but license references. You could say "valid SPDX license expressions".

Reply:
Done.